wekesa931/Questioner
# [finishes #163064894] creates the sign-up api endpoint]

#### What does this PR do?
It creates the sign-up api endpoint
#### Description of Task to be completed?
To create the sign up endpoint path:
```
POST         /api/v1/user/auth/signup
```
This endpoint captures the required details of the user which are:
```
{
    “id” : Integer ,
    “firstname” : String ,
    “lastname” : String ,
    “othername” : String ,
    “email” : String ,
    ”phoneNumber” : String,
    “username” : String ,
    “registered” : Date ,
    “isAdmin” : Boolean ,

}
```
#### How should this be manually tested?
clone the repository to your local machine
```
$ git clone https://github.com/wekesa931/Questioner.git
```
Set up the virtual environment
```
$ pip install virtualenv
$ virtualenv venv
$ venv source bin/activate
```
run the application
```
$ python run.py
```
Test the endpoint in postman as follows:
1. In postman, go to path
```
 POST      /api/v1/user/auth/signup
```
2. Add the details of the user
3. Check the user has been added with the GET method
```
GET         /api/v1/users
```

#### What are the relevant pivotal tracker stories?
    #163064894
#### Screenshots (if appropriate)
![signup](https://user-images.githubusercontent.com/31995387/50879098-b87d3880-13ea-11e9-8ec7-d532f0a968df.png)
